package.json

```diff
@@ -14,6 +14,11 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@vx/gradient": "0.0.136",
+    "@vx/group": "0.0.136",
+    "@vx/mock-data": "0.0.136",
```

Comment:
It's fine for now. For sure, we'll need to remove when the feature is fully fleshed out.